standard_logger.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+def login():
```

Comment:
Yes i created a wrapper module because it then we would not have to write logger commands again and again.